src/externalized/AmmoTypeModel.cc

```diff
@@ -7,9 +7,9 @@
 #include "JsonObject.h"
 
 AmmoTypeModel::AmmoTypeModel(uint16_t index_,
-                             const char* internalName_
-  )
-  :index(index_), internalName(internalName_)
+				const char* internalName_
+)
+	:index(index_), internalName(internalName_)
 {
```

Comment:
I hate whitespace after a bracket.

This is the closes we can get without mixing spaces and tabs, so I would like to keep it as it is.